catalog/imaging/imaging-app/src/main/resources/features.xml

```diff
@@ -16,6 +16,12 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <feature name="imaging-thumbnail-plugin" install="auto" version="${project.version}"
+             description="The imaging thumbnail plugin populates thumbnails on nitf metacards.">
```

Comment:
This plugin produces 3 images: thumbnail, overview and original.  I'd suggest renaming it to something like 'imaging-render-plugin' and updating the description to include that.